lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -196,8 +188,13 @@ func prefixImageID(imageID string) string {
 	return "sha256:" + imageID
 }
 
-// AddImage adds an image to the image cache
-func (ic *ICache) AddImage(imageConfig *metadata.ImageConfig) {
+// Add adds an image to the image cache
+func (ic *ICache) Add(imageConfig *metadata.ImageConfig, save bool) {
+	defer trace.End(trace.Begin(""))
```

Comment:
looks like you have add with save or not, but do not have delete with save option. Is this option really necessary?